lms/views/admin/application_instance.py

```diff
@@ -13,6 +14,35 @@
 from lms.views.admin import flash_validation
 from lms.views.admin._schemas import EmptyStringInt
 
+# Helper to declare settings as secret
+AES_SECRET = object()
+APPLICATION_INSTANCE_SETTINGS = {
+    ("blackboard", "files_enabled"): asbool,
```

Comment:
Not sure if having them here in the "groups.setting" string format would be less work than the tuples.

If you are already thinking of maybe moving this to the application settings model probably not worth thinking to much about it.